README.md

```diff
@@ -130,6 +130,18 @@ GDS::SSO.config do |config|
 end
 ```
 
+If you are using a Rails 5 app in
```

Comment:
Would be tempted to put this under a heading so it's easier to notice.

Reply:
I think it's probably ok with a light touch as it's something that should **just work ™**